ginkgo_ai_client/queries.py

```diff
@@ -494,21 +494,20 @@ class RNADiffusionMaskedQuery(QueryBase):
     def to_request_params(self) -> Dict:
 
         data = {
-            "three_utr": self.three_utr.replace(
-                "<mask>", "[MASK]"
-            ),  # UTR tokenizers require [MASK] but api client accepts <mask> for consistence across models
-            "five_utr": self.five_utr.replace("<mask>", "[MASK]"),
+            # Many people in the field use [MASK] but our API client uses <mask> for all models
+            "three_utr": self.three_utr.replace("[MASK]", "<mask>"),
```

Comment:
I think we should just remove `replace("[MASK]", "<mask>")` altogether and only allow `<mask>`

Reply:
Done, I removed the `replace()`s